pyproject.toml

```diff
@@ -1,17 +1,78 @@
 [build-system]
-requires = ["jupyter_packaging~=0.10,<2", "jupyterlab~=3.6"]
-build-backend = "jupyter_packaging.build_api"
+requires = ["hatchling>=1.22.0", "jupyterlab>=4.0.0,<5", "hatch-nodejs-version>=0.3.2"]
+build-backend = "hatchling.build"
 
-[tool.jupyter-packaging.options]
-skip-if-exists = ["jupyter_server_extension/labextension/static/style.js"]
-ensured-targets = ["jupyter_server_extension/labextension/static/style.js", "jupyter_server_extension/labextension/package.json"]
+[project]
+name = "maap_jupyter_server_extension"
+readme = "README.md"
+license = { file = "LICENSE" }
+requires-python = ">=3.8"
+classifiers = [
+    "Framework :: Jupyter",
+    "Framework :: Jupyter :: JupyterLab",
+    "Framework :: Jupyter :: JupyterLab :: 4",
+    "Framework :: Jupyter :: JupyterLab :: Extensions",
+    "Framework :: Jupyter :: JupyterLab :: Extensions :: Prebuilt",
+    "License :: OSI Approved :: BSD License",
+    "Programming Language :: Python",
+    "Programming Language :: Python :: 3",
+    "Programming Language :: Python :: 3.8",
+    "Programming Language :: Python :: 3.9",
+    "Programming Language :: Python :: 3.10",
+    "Programming Language :: Python :: 3.11",
+]
+dependencies = [
+    "jupyter_server>=2.0.1,<3"
```

Comment:
@marjo-luc Can we make this `"jupyter_server>=2.0.1,<2.13.0"`? There was an update in 2.13.0 `jupyter_server` that caused the extension to break
https://github.com/MAAP-Project/maap-workspaces/pull/94
I believe the error was that a new function was added to `jupyter_server` with the same name of a function we had written ourselves and that caused problems

Reply:
Yes -- let me go ahead and update all the extensions with this.

Reply:
Actually, looks like this was the only extension with that dependency. Update pushed.